app/build.gradle

```diff
@@ -13,13 +13,31 @@ android {
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
 
+    signingConfigs {
+        debug {
+            // keytool -genkeypair -v -keystore my-debug-key.keystore -alias my-key-alias -keyalg RSA -keysize 2048 -validity 10000
+            storeFile file('my-debug-key.keystore')
+            storePassword '123456'
```

Comment:
@xinminsu Since this is an open source project, we cannot put the keystore password and key password in the code itself. Do you have a solution for fetching these values as command line arguments?